lib/src/components/NxTag/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {ReactNode, WeakValidationMap} from 'react';
+import * as PropTypes from 'prop-types';
+
+export interface Props {
+  onTagSelect?: (() => void) | null;
+  tagSelected?: boolean;
+  children: ReactNode;
+  tagColor?: string | null;
+  className?: string | null;
+}
+export const propTypes: WeakValidationMap<Props> = {
+  onTagSelect: PropTypes.func,
+  tagSelected: PropTypes.bool,
+  children: PropTypes.string,
```

Comment:
This is overly constrained.  I believe there is a `PropTypes.node` which corresponds to the `ReactNode` typescript type.  Also needs `.isRequired`

Reply:
> This is overly constrained. I believe there is a `PropTypes.node` which corresponds to the `ReactNode` typescript type. Also needs `.isRequired`

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/58a77feeff76fb9ca507d41a4693aa3162949e0e